Dockerfile

```diff
@@ -245,8 +245,8 @@ RUN chmod +x ./download-machine.sh ./download-machine-celestia.sh
 RUN ./download-machine.sh consensus-v30 0xb0de9cb89e4d944ae6023a3b62276e54804c242fd8c4c2d8e6cc4450f5fa8b1b && true
 RUN ./download-machine.sh consensus-v31 0x260f5fa5c3176a856893642e149cf128b5a8de9f828afec8d11184415dd8dc69
 RUN ./download-machine.sh consensus-v32 0x184884e1eb9fefdc158f6c8ac912bb183bf3cf83f0090317e0bc4ac5860baa39
-RUN ./download-machine-celestia.sh v3.2.1-rc.1 0xe81f986823a85105c5fd91bb53b4493d38c0c26652d23f76a7405ac889908287 celestiaorg
```

Comment:
I think we need ./download-machine-celestia.sh

Reply:
[d740d0f](https://github.com/EspressoSystems/nitro-espresso-integration/pull/610/commits/d740d0f297a6c1d18fd501574080335b918d259d)

Reply:
Upstream celestia actually uses the regular download machine script with the secondary argument to specify their GitHub org. I can link it in another comment, but I’m on my phone rn.